r/src/differential_expression.r

```diff
@@ -0,0 +1,193 @@
+
+# bh.adjust function from [1]
+#' @description BH P-value adjustment with a log option. 
+#' Helper functions for pagoda2 DE pipeline. 
+bh.adjust <- function (x, log = FALSE) {
```

Comment:
Thanks! For the next iteration with the data-ingest multisample script I will measure the speed of both functions :). The reason I used this funcitonis  because is the one that is included in the pagoda2 DE pipeline.